Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM docker:stable
+
+RUN set -eux ; \
+    apk add  --no-cache bash
+COPY entrypoint.sh /usr/bin/entrypoint
```

Comment:
I guess there's no way to define new github actions with multiple steps without using a Docker container?